### DISCUSSION TITLE
Setting a default field variant?

### DISCUSSION THREAD

**Discussion by abea** [vuetifyjs/vuetify#16891] [Q&A] [answered]
We're upgrading from Vue 2/Vuetify 2 right now. Our existing design has text fields that look like the variant `'underlined'`, not the default styling. It seems like we can either set the `variant` prop on every field in our code base or override styles with our own (maybe needing `!important`?).

It's possible that we could override SCSS variables to get as close as possible, but is there a way to simply tell Vuetify (v3) that we want the `underlined` variant to be the default?

**Comment by KaelWD** [comment, marked as answer]
https://vuetifyjs.com/en/features/global-configuration/

**Reply by abea** [reply]
Perfect. Thank you.

**Comment by theking2** [comment]
But how do I set boht VTextField _and_ VTextarea without repeating?

**Reply by abea** [reply]
Both are based on VInput. Some props of those two might be inherited directly such that you could set the property on VInput. That said, we do have separate global settings for VTextField, VTextarea, VAutocomplete, and a couple others. I'm not sure if that's because we tried combining them and it didn't work.

**Reply by theking2** [reply]
Just for my understanding setting globals on VInput will automagically propagete (inherited) derived input elements?

**Reply by abea** [reply]
That works for many inheritance situations, yes. Some components may have internal overrides that replace the inherited prop, so you might need to apply it directly to the ~~forehead~~ child component.

**Reply by theking2** [reply]
Thanks that answers my question!
